spec/forms/candidate_interface/degree_enic_form_spec.rb

```diff
@@ -65,4 +65,41 @@
       end
     end
   end
+
+  describe 'ENIC reference validations' do
+    let(:form) {
+      described_class.new(
+        degree: degree,
+        have_enic_reference: 'yes',
+        enic_reference: number_of_characters,
+        comparable_uk_degree: 'bachelor_ordinary_degree',
+      )
+    }
+
+    let(:degree) {
+      create(
```

Comment:
```suggestion
      build(
```
I think you can get away without saving the degree record.

Reply:
thanks